packages/xrpl/src/models/methods/subscribe.ts

```diff
@@ -454,9 +454,20 @@ export interface PathFindStream extends BaseStream {
       }
 }
 
+/**
+ * The manifest stream sends messages when a new manifest has been processed or updated.
+ * The message contains the public key, either the master public key or ephemeral public key.
+ *
+ * @category Streams
+ */
+export interface ManifestStream extends BaseStream, ManifestRequest {
+  type: 'manifestReceived'
```

Comment:
I could not find the documentation for the `ManifestStream` in this [docs](https://xrpl.org/docs/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe) page.

Where did you come across this `manifestRecieved` keyword? Are you using this in your application?

Reply:
This stream was already in the codebase, but was not typed. We can do a deep drive to figure out why this was here in the first place, but I figure we should at least type something here and avoid the usage of index signatures.

We can remove this addition if its going to create a fuss.

Reply:
I found this [reference](https://github.com/XRPLF/xrpl.js/blob/7fe27f3d9c8cdb92a7f9ad97daa9cf7bb0344637/packages/xrpl/src/models/methods/subscribe.ts#L477) to `manifestReceived` event trigger. Furthermore, that file uses the `ManifestRequest` type. Do you still see the need for `ManifestStream` type ?

Reply:
ignore the previous comment. Yes, I see the need for the `ManifestStream` type.

Reply:
I found a relevant [request-for-feature](https://github.com/XRPLF/xrpl-dev-portal/issues/958) issue in the XRPL docs repository.